lib/github_changelog_generator.rb

```diff
@@ -139,7 +139,7 @@ def fetch_merged_at_pull_requests
       }
 
       if @options[:verbose]
-        puts 'Fetching merged dates... Done!'
+        puts "Fetching merged dates: Done!"
```

Comment:
@tuexss it really makes sense. I just new in ruby, and consider, if RubyMine and rubocop pronouncedly highlight this - it should be like that "a priori". So, I thinking to move to double quote and revert this "fancy" PR :)

.. and add to rubocom.yml.

    Style/StringLiterals:
      EnforcedStyle: double_quotes

Thanks, for point it out!